wazuh_ossec/help.md

```diff
@@ -603,7 +607,8 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
-* 1.0.1 - New spec and help.md format for the Hub | Spelling fixes in descriptions
+* 2.0.0 - Removed colon from variable names wazuh-modules_oscap and wazuh-modules_database
+* 1.0.1 - New spec and help.md format for the Hub
```

Comment:
Latest one is https://market.komand.com/plugins/rapid7/wazuh_ossec/1.0.0 so 1.0.1 was never released.

Combine this line with 2.0.0 and release the major version of 2.0.0